test/framework/github.py

```diff
@@ -410,16 +406,17 @@ def test_github_fetch_easyblocks_from_pr(self):
             'pr_target_account': gh.GITHUB_EB_MAIN,
         })
 
+        # TODO: no 5.x PRs for new easyblocks
```

Comment:
Once the `5.0.x` branch is collapsed into `develop`, this should no longer be a problem?